Mandelbrot.hs

```diff
@@ -10,11 +13,19 @@ import Data.Colour.SRGB
 import Data.Complex
 import Data.List
 import Data.Tuple.Extra
+import Options.Generic
+
+data Opts = Opts
+    { out :: FilePath
+    , width :: Int
+    , height :: Int
+    , xMin :: Double
+    , xMax :: Double
+    , yMin :: Double
+    , yMax :: Double
+    }
+    deriving (Eq, Ord, Show, Generic, ParseRecord)
```

Comment:
The changes in this file don't really in principle have anything to do with the purpose of this branch...